src/lib/mina_base/test/zkapp_command/zero_vesting_period.ml

```diff
@@ -0,0 +1,421 @@
+open Core_kernel
+open Mina_base
+open Snarky_backendless
+open Snark_params.Tick
+
+let%test_module "zero_vesting" =
+  ( module struct
+    let mk_zkapp_with_vesting_period n =
```

Comment:
Is it possible to generate this message using some test utilities like in https://github.com/MinaProtocol/mina/blob/develop/src/lib/network_pool/transaction_pool.ml#L2779 instead of pasting recorded message ?

Reply:
It's possible to do something like that, but since we have an example that exercises the check, it's simpler to use that.